### DISCUSSION TITLE
Add "Check latest release on GitHub" button on about tab

### DISCUSSION THREAD

**Discussion by xlionjuan** [rustdesk/rustdesk#5475] [Feature Request] [answered]
I know you have no time to implement auto update function, so I think you can add a button called "Check latest release on GitHub" 
And point it to https://github.com/rustdesk/rustdesk/releases/latest
so we can check the update more easily.

**Comment by rustdesk** [comment, marked as answer]
Actually, we have, but I do not like its implementation, so turned it off.

https://github.com/rustdesk/rustdesk/blob/115221098a96a1bb9fa092d94d20ab30a8a9ecb5/src/common.rs#L836

@sahilyeole Please rewrite this to get version like this from github api.

https://gist.github.com/lukechilds/a83e1d7127b78fef38c2914c4ececc3c

or 

https://stackoverflow.com/questions/29109673/is-there-a-way-to-get-the-latest-tag-of-a-given-repo-using-github-api-v3

**Reply by beelchester** [reply]
Surely 👍
